A32NX/html_ui/Pages/A32NX_Core/A32NX_BaroSelector.js

```diff
@@ -0,0 +1,50 @@
+// Used to determine if an ICAO code uses InHG or HPa.
+function AirportUsesHPA(icao) {
+    const InHgRegions = ["K", "C", "M", "P", "RJ", "RO", "TI", "TJ"];
+    for (let i = 0; i < InHgRegions.length; i++)
+        if (icao.substr(0, InHgRegions[i].length) == InHgRegions[i]) return false;
+    return true;
```

Comment:
Can be simplified to this: 

```suggestion
    return !InHgRegions.some(r => icao.startsWith(r));
```